lib/Settings/Display.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:cboard_mobile/shared/app-bar.dart';
+import 'package:flutter/material.dart';
+
+class Display extends StatelessWidget {
+// static const String route = "display";
+// final Map<String,String> arguments;
+// Display(this.arguments);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: BaseAppBar(
+          title: Text('Display'),
+          appBar: AppBar(),
+        ),
+        body: Center(
+          child: MyStatefulWidget(),
+        ));
+  }
+}
+
+enum size { Standard, Large, ExtraLarge }
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
+  bool _darkTheme = false;
+  bool _predictiveText = false;
+  bool _outputBar = false;
+  size _uiSize = size.Standard;
+  size _fontSize = size.Standard;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
+      children: <Widget>[
+        radio_section('UI Size', 'Elements Size', _uiSize),
+        divider(),
+        radio_section('Font Size', 'App Font Size', _fontSize),
+        divider(),
+        listTile_section(
+            'Enable Dark Theme',
+            'The theme specifies the color of the components, darkness of the surfaces, level of shadow, appropriate opacity of ink elements, etc.',
+            _darkTheme),
+        divider(),
+        listTile_section(
+            'Enable Predictive Text',
+            'Predictive text will arrange the cards so that recommended cards based on past history will be recommended and shown first',
+            _predictiveText),
+        divider(),
+        listTile_section(
+            'Hide the Output Bar',
+            'Hides the white bar on the top where you build a sentence',
+            _outputBar),
+      ],
+    );
+  }
+
+  Widget divider() {
+    return Divider(
+      height: 20,
+      color: Color(0xffE0E0E0),
+      thickness: 1,
+    );
+  }
+
+  Widget listTile_section(String title, String subtitle, bool state) {
```

Comment:
Do you think you can modify this widget so that it extends `Stateless/StatefulWidget`? I'm not too sure of the different it makes, but that class extension might have additional features that we can use.

Can we also use CamelCase naming conventions to keep things consistent too?

You can also even export this widget into your own `shared` folder so others can access it if they need to also!